javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -324,19 +182,19 @@
                     "version_added": "25"
                   },
                   "edge": {
-                    "version_added": "14"
+                    "version_added": "12"
                   },
                   "firefox": {
-                    "version_added": "52"
+                    "version_added": "29"
                   },
                   "firefox_android": {
                     "version_added": "56"
                   },
                   "ie": {
-                    "version_added": false
+                    "version_added": "11"
                   },
                   "nodejs": {
-                    "version_added": "4.0.0"
+                    "version_added": "0.12.0"
```

Comment:
FYI slight exception to "copying the parent compatibility". This copies the parent first introduction when things were "partial". This must be correct because there are options that have versions before v13.